modules/mesh-layers/src/scenegraph-layer/scenegraph-layer-vertex.glsl.js

```diff
@@ -66,7 +67,13 @@ void main(void) {
     geometry.uv = pbr_vUV;
   #endif
 
-  vec3 pos = (instanceModelMatrix * (sceneModelMatrix * POSITION).xyz) * sizeScale + instanceTranslation;
+  float sizeMinPixels = sizePixels.x;
+  float sizeMaxPixels = sizePixels.y;
+  float sizePixelsDimension = sizePixels.z;
+  float originalSize = project_size_to_pixel(sizePixelsDimension * sizeScale);
```

Comment:
The argument of this function must be in common space. i.e. not "pixels"

Reply:
New logic should fix it.